api/src/scripts/populate_db_test_data.py

```diff
@@ -113,7 +115,16 @@ def populate(self):
         Populate the database with the test data
         """
         self.logger.info("Populating the database with test data")
-        self.populate_test_datasets()
+
+        if not self.filepaths:
+            self.logger.error("No file paths provided")
+            return
+        if not isinstance(self.filepaths, list):
+            self.filepaths = [self.filepaths]
```

Comment:
This conversion should be done in the _init function to avoid _confusion_ of use the filepathps property within the class; even though  in this case we only have one function that depends on it.